multiraft/multiraft.go

```diff
@@ -1039,14 +1046,11 @@ func (s *state) handleWriteResponse(response *writeResponse, readyGroups map[uin
 				}
 				continue
 			case raftpb.MsgHeartbeatResp:
-				if _, ok := noMoreHeartbeats[msg.To]; ok {
-					if log.V(7) {
-						log.Infof("node %v dropped redundant heartbeat response to node %v",
-							s.nodeID, msg.To)
-					}
-					continue
+				if log.V(7) {
```

Comment:
if sending/receiving coalesced heartbeats is `V(7)`, this should be higher (or omitted - we don't care about per-group heartbeats any more at all).